androidApp/src/main/java/dev/johnoreilly/confetti/ConfettiApplication.kt

```diff
@@ -32,7 +30,10 @@ class ConfettiApplication : Application(), ImageLoaderFactory {
         } else {
             Firebase.crashlytics.setCrashlyticsCollectionEnabled(false)
         }
-        
+
+        // Initialize Logging.
+        Napier.base(DebugAntilog())
```

Comment:
Does something equivalent have to be done for Wear and iOS clients/

Reply:
For iOS:
```
fun debugBuild() {
    Napier.base(DebugAntilog())
}
```

We would also need to do it on WearOS + Compose Desktop.

Reply:
I will include it on Wear and Desktop, I don't have iOS tooling to compile and test, tho.

Reply:
Included on Android, Wear, Desktop. I currently don't have the tooling to compile and test iOS, someone will need to do it there. 😄